hudi-integ-test/src/test/java/org/apache/hudi/integ/testsuite/TestDFSHoodieTestSuiteWriterAdapter.java

```diff
@@ -52,6 +52,9 @@
 import org.junit.jupiter.api.Test;
 import org.mockito.Mockito;
 
+/**
+ * An adapter of {@link HoodieTestSuiteWriter} help to test write DFS file.
```

Comment:
{@link HoodieTestSuiteWriter}. Helps to test writing a DFS file.

Reply:
done